### DISCUSSION TITLE
How to get validated fields

### DISCUSSION THREAD

**Discussion by faydanube** [livewire/livewire#2143] [1. Help] [answered]
A field that has been validated can be retrieved in a laravel document by using the request()->validated() method. 
How do I retrieve this in livewire? 
This was useful for my project, I tried request() and $this->request() and neither of them worked.

Thanks a lot.

**Comment by ju5t** [comment, marked as answer]
https://laravel-livewire.com/docs/2.x/input-validation is everything you need. This will work exactly as you would expect from standard Laravel validation too.

You add rules:

```
protected $rules = [
    'email' => 'required|email',
];
```

And then whenever you need to validate:

```
$validatedData = $this->validate();
```

Which would give you the validated data.

**Reply by faydanube** [reply]
thanks
